tensorflow/python/ops/image_ops_impl.py

```diff
@@ -640,6 +640,22 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example:
+  
+  >>> image = [[[1.0, 2.0, 3.0],[4.0, 5.0, 6.0]],...
```

Comment:
Can you use integer input values here?

Also, use a 2x2 matrix (here and everywhere else).

Reply:
> Can you use integer input values here?
> 
> Also, use a 2x2 matrix (here and everywhere else).

This can only be done with `tf.image.transpose()` and `tf.image.adjust_hue()`
all the others will raise ValueErrors